preauth-simulation-filter-servlet_2.5/pom.xml

```diff
@@ -0,0 +1,54 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>com.github.nblair</groupId>
+		<artifactId>developer-tools</artifactId>
+		<version>0.1.1-SNAPSHOT</version>
+	</parent>
+	<artifactId>preauth-simulation-filter-servlet_2.5</artifactId>
+	<name>Pre-Authentication Simulation Filter</name>
+	<description>Servlet Filter useful for simulating Pre-Authentication environment in a servlet 2.5 environment. This allows a developer to spoof request headers or environment variables that would provided by preauth tools like Shibboleth and Siteminder without having those installed locally.</description>
+	
+	<dependencies>
+		<dependency>
+			<groupId>javax.servlet</groupId>
+			<artifactId>servlet-api</artifactId>
+			<version>2.5</version>
+		</dependency>
+		<dependency>
+			<groupId>org.apache.commons</groupId>

```

Comment:
Could we drop the commons-lang3 dependency? It looks like it's used only on one line in PreAuthenticationSimulationServletFilter, i'd rather not have these modules pull in extra dependencies.